_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -9,6 +9,12 @@ This section will be updated prior to the next release.
 Global parameters now support the same types as smart variables and smart class parameters.
 As a part of this change searching by parameter value is no longer allowed. ([#4127](https://projects.theforeman.org/issues/4127))
 
+#### Smart Proxy Exposed Capabilities and Settings
+
+The Smart Proxy now provides a v2 features api allowing plugins to declare capabilities and expose settings from their configuration.
+The Foreman server has been updated to use this new api and store the declared capabitlities and settings.  These will be used in the
+future to enhance and simplify our interactions with the Smart Proxy in the future.
```

Comment:
in the future... in the future

Reply:
also, perhaps link to the redmine issue(s) or discourse discussion so people interested in more information can find it?

Reply:
We can add a link to https://github.com/theforeman/theforeman.org/pull/1342 once it's finished and published.